package.json

```diff
@@ -65,6 +65,7 @@
     "moment": "^2.24.0",
     "morgan": "^1.9.1",
     "multer": "^1.4.1",
+    "open": "^6.4.0",
```

Comment:
Can you explain to me the use of this package? @MCFrank16

Reply:
this package open up the browser, I used it in the middleware to open up the browser with one of the URL I declared inside there..